### PR TITLE
changes made in logo and index.html file

### PR DESCRIPTION
**Pull request by rajataaron**
Changes made in logo color of toggle button and added some space into section.

### CODE REVIEW

File: css/custom.css [hunk 1]
```diff
@@ -103,6 +103,10 @@ section {
     margin-bottom:30px;
 }
 
+.section-head{
```
Comment (CuriousLearner) [inline css/custom.css]: Leave a space between selector and starting braces

File: css/custom.css [hunk 2]
```diff
@@ -197,13 +201,16 @@ section {
 }
 
 .navbar-brand img{
```
Comment (CuriousLearner) [inline css/custom.css:196]: Same here.

**Review by CuriousLearner — CHANGES_REQUESTED** [review]

File: css/custom.css [hunk 3]
```diff
@@ -103,6 +101,10 @@ section {
     margin-bottom:30px;
 }
 
+.section-head{
+    margin-top: 50px;
```
Comment (theskumar) [inline css/custom.css]: This is going to affect both the mobile and desktop view. Please make sure this only on desktop view. :)

**Review by theskumar — COMMENTED** [review]
Minor feedback and rest look good.

**Review by theskumar — APPROVED** [review]

**Comment by akshayaurora** [comment]
@rajataaron 👍 thanks

**Pull request merged**